tests/conftest.py

```diff
@@ -89,7 +89,7 @@ def pytest_configure(config: pytest.Config) -> None:
     # Set 'cgroup_version' config value.
     try:
         output = ctr_client.run(
-            "ubuntu:20.04",
+            "ubuntu:24.04",
```

Comment:
These tests were deliberately written to test systemd in ubuntu:20.04. I'm not currently using the tests, but wouldn't want to remove support for testing 20.04 right now, so would only accept support for 22.04 and 24.04 if there's a clean way to support multiple at once (perhaps chosen via a CLI arg).

Reply:
Ubuntu 20.04 will be EOL in 7 months, so I don't think it makes sense to waste resources for keeping this. At least I personally do not plan to spend time on this.

Reply:
That's okay, I'm just letting you know that I'm not really open to accepting this change (and the associated changes such as requiring `/run/lock` that we don't fully understand) since my use-case hasn't currently moved on from ubuntu20.04 (it's the systemd version that's more relevant than the distro version, so the ubuntu EOL doesn't particularly bother me).